### PR TITLE
templates: update record detail

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
@tiborsimko

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio_records_ui/templates/invenio_records_ui/detail.html [hunk 1]
```diff
@@ -54,20 +54,63 @@
...
-  {%- block record_body %}
-  {% if record %}
-  <div class="panel panel-default">
-    <ul class="list-group">
-      {{ record_content(record) }}
-    </ul>
+  <div class="extra-info">
+    {%- if pid %}
+      {%- block record_control_number %}
+      <b>{{ _('control number:') }}</b>
```
Comment (lnielsen) [inline invenio_records_ui/templates/invenio_records_ui/detail.html]: control number -> Control number ?

Comment (jirikuncar) [inline invenio_records_ui/templates/invenio_records_ui/detail.html]: `{{ _('Persistent identifier') }}` unless we use `record['control_number']`

**Comment by jirikuncar** [comment]
IMHO we should not update this template but instead override it in `invenio-marc21`.

**Comment by lnielsen** [comment]
@jirikuncar Fully agree - didn't notice before that the PR was actually on records-ui

**Comment by jirikuncar** [comment]
Issue moved to [inveniosoftware/invenio-marc21 #20](https://github.com/inveniosoftware/invenio-marc21/issues/20) via [**ZenHub.io**](https://www.zenhub.io/)

**Pull request closed without merging**